docs/docs/glossary.md

```diff
@@ -3,44 +3,80 @@ title: Glossary
 tags: [protocol, glossary]
 ---
 
+import { Glossary } from '@site/src/components/Snippets/glossary_snippets';
+
+### Aztec
+
+<Glossary.Tools.aztec />
+
+### Aztec Wallet
+
+<Glossary.Tools.aztec_wallet />
+
```

Comment:
I think it would be good to add a link to the `aztec-wallet` reference page for more info. docs/docs/developers/reference/environment_reference/cli_wallet_reference.md

Reply:
Done.